docs/admin_api/rooms.md

```diff
@@ -475,16 +524,82 @@ The following JSON body parameters are available:
 
 The JSON body must not be empty. The body must be at least `{}`.
 
+## Status of deleting rooms
+
+**Note**: This API is new, experimental and "subject to change".
+
+It is possible to query the status of the background task for deleting rooms.
+The status can be queried up to 24 hours after completion of the task,
+or until Synapse is restarted (whichever happens first).
+
+With this API you can get the status of all tasks the last 24h for this `room_id`.
+If you want only get the status of one specific task, you can also use
+the [Purge status query API](purge_history_api.md#Purge-status-query).
```

Comment:
It feels confusing to tell users to use the *purge* status query for the results of a *delete* operation. I think if this is a thing we need, we should add a `GET /_synapse/admin/v2/rooms/<room_id>/delete_status/<purge_id>` for it.

Reply:
What is with this: `GET /_synapse/admin/v2/rooms/delete_status/<purge_id>`?
Than you can request the status with `purge_id` and do not need the `room_id`.

Reply:
> What is with this: `GET /_synapse/admin/v2/rooms/delete_status/<purge_id>`?

sounds fine.